### DISCUSSION TITLE
Route is *.test instead of localhost?

### DISCUSSION THREAD

**Discussion by philliphemleb** [tighten/ziggy#474] [Q&A] [answered]
Hey,

I am using the route method like that `:href="route('dashboard_index')"` but the url is then "project_a.test/dashboard" instead of "localhost/dashboard".

I am using Laravel Sail, my tests are working fine in my docker container. So the laravel route´s helper gives me "localhost" when its supposed to do, how can I get the exact same behaviour with the ziggy route helper?

Thanks for answers, and also nice work and thanks for the great framework!

**Comment by bakerkretzmar** [comment]
What's the actual URL in the address bar on the page where you're seeing that error, and what's your `APP_URL`?

**Comment by philliphemleb** [comment, marked as answer]
I did a massive mistake by not changing the APP_URL. But thanks for your really fast reply! I am using the route helper for my navigation, and I think its working now.
